sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -390,7 +395,8 @@ void wait_enough_for_alert_ping() {
   } else if (kDeviceType == kDeviceSimDV) {
     // NUM_ALERTS*2*margin_of_safety*(2**DW)*(1/kClockFreqPeripheralHz)
     // (2**6)*2*4*(2**3)*(40ns) = 160us
-    busy_spin_micros(160);
+    flash_ctrl_testutils_backdoor_wait_eq(&flash_ctrl, (uintptr_t)&kTestIp,
+                                          peri_idx + 1, 160);
```

Comment:
Unfortunately, although everything seems to be right, this does not work as expected. To overcome `sw_backdoor_mechanism's` caching issue, I tried to use a shadow copy of the polling variable `kTestIp`, and it seems to be working. Here is the code that is working:

```suggestion
 //flash_ctrl_testutils_backdoor_wait_eq(&flash_ctrl, (uintptr_t)&kTestIp,
    //                                      peri_idx + 1, 640);
    const ibex_timeout_t timeout_ = ibex_timeout_init(640);
    while((kTestIp != peri_idx+1) | (kTestIp2 != peri_idx+1)) {
    //TODO: Remove after debugging.	
    LOG_INFO("ktestip=%d, ktestip2=%d", kTestIp, kTestIp2);
        // Check timeout.
    CHECK(!ibex_timeout_check(&timeout_),
          "Timed out after %d usec (%d CPU cycles) waiting for exp_data = %d",
          640, (uint32_t)timeout_.cycles, peri_idx+1);
    }
    LOG_INFO("Out of the SV polling loop");
```